pycti/connector/opencti_connector_helper.py

```diff
@@ -1564,11 +1566,11 @@ def send_stix2_bundle(self, bundle: str, **kwargs) -> list:
         entities_types = kwargs.get("entities_types", None)
         update = kwargs.get("update", False)
         event_version = kwargs.get("event_version", None)
-        bypass_split = kwargs.get("bypass_split", False)
```

Comment:
@richard-julien actually it's a breaking change, not a deprecation. Having this option to True won't work as expected anymore.

Reply:
I will update the comment. Actually the split would be done at worker level anyway, there is no way to bypass the split ultimately.